### DISCUSSION TITLE
Minecraft Bedrock server for Docker arm64 (Odroid C4)

### DISCUSSION THREAD

**Discussion by makakam83** [itzg/docker-minecraft-bedrock-server#209] [General]
Hi
I'd like to install Minecraft Bedrock server on my Odroid C4 in a Docker container.
Is such version available? I can't find it. Thanks a lot.

**Comment by itzg** [comment]
Unfortunately Mojang only provides an x86 download for the bedrock server.

**Reply by tehstu** [reply]
I'm learning Docker and looking for interesting projects to tinker with. This looked perfect and I got quite a ways into considering the configuration until I realized, reading the comment in Dockerfile, that it was amd64 only. Unfortunate, but hopefully mojang will do something now we have the Apple M1.

**Reply by DominusExult** [reply]
Due to Rosetta2 this actually runs on an Apple M1. No idea how much power is wasted, though.

**Comment by rshurts** [comment]
I don't have an arm device to try this on, but has anyone tried doing a multi-platform build, `docker buildx build --platform linux/arm/v7 .` and running that image on an arm server?

**Reply by itzg** [reply]
Unfortunately the multi-platform support there doesn't ultimately fix it. (BTW the Java edition server image is using multi-platform builds...because Java). 

It would end up being an ARM image, with ARM OS libraries, etc, with an x86 bedrock executable. 

When I get more info on https://github.com/itzg/docker-minecraft-bedrock-server/issues/140#issuecomment-1113781909 then I can look into trying to publish an image based on that.
